bxserum/provider/common.go

```diff
@@ -30,9 +33,10 @@ type RPCOpts struct {
 	Endpoint   string
 	Timeout    time.Duration
 	PrivateKey *solana.PrivateKey
+	AuthHeader string
 }
 
-func DefaultRPCOpts(endpoint string) RPCOpts {
+func DefaultRPCOpts(endpoint string, authHeader string) RPCOpts {
```

Comment:
better to load `authHeader` from the environment variable in this function, to save lines throughout the rest of the changeset